src/main/java/ca/bc/gov/chefs/etl/forms/pcd/chc/pcPatientServices/json/Root.java

```diff
@@ -0,0 +1,97 @@
+package ca.bc.gov.chefs.etl.forms.pcd.chc.pcPatientServices.json;
```

Comment:
It looks like your IDE is set for 2 spaces. Existing code is all 4 spaces/tabs.

Reply:
I updated all the files to 4 spaces and re-linted them.